RePoE/parser/util.py

```diff
@@ -80,3 +80,19 @@ def ignore_mod_domain(domain):
         MOD_DOMAIN.AFFLICTION_JEWEL,
     }
     return domain not in whitelist
+
+
+def find_missing_stat_descriptions(file_system, data_path, relational_reader, translation_file_cache, ot_file_cache):
+    node = file_system.build_directory()
+    keys = list(node["Metadata"]["StatDescriptions"].children.keys())
+
+    missing_keys = []
+    for key in keys:
+        if "descriptions.txt" in key:
+            if key not in [written_file[0] for written_file in WRITTEN_FILES]:
+                missing_keys.append(key)
+    return missing_keys
+
+
+if __name__ == "__main__":
+    print(call_with_default_args(find_missing_stat_descriptions))
```

Comment:
I don't really like having this check as the only thing that's done when running a file called `util.py`. How about asserting on find_missing_stat_descriptions returning an empty list in `modules/stat_translations.write` (or raising a ValueError as is done in `base_items`)?